macstadium-pod-1/workers.tf

```diff
@@ -0,0 +1,272 @@
+variable "latest_travis_worker_version" {}
+
+variable "travis_worker_version" {
+  default = "v3.4.0"
+}
+
+data "template_file" "worker_config_common" {
+  template = <<EOF
+export TRAVIS_WORKER_BUILD_FIX_ETC_HOSTS="true"
+export TRAVIS_WORKER_BUILD_FIX_RESOLV_CONF="true"
+export TRAVIS_WORKER_BUILD_PARANOID="false"
+export TRAVIS_WORKER_QUEUE_TYPE="amqp"
+export TRAVIS_WORKER_QUEUE_NAME="builds.macstadium6"
+export TRAVIS_WORKER_PROVIDER_NAME="jupiterbrain"
+export TRAVIS_WORKER_JUPITERBRAIN_IMAGE_SELECTOR_TYPE="api"
+export TRAVIS_WORKER_JUPITERBRAIN_SSH_KEY_PATH="/etc/travis-vm-ssh-key"
+EOF
+}
+
+module "worker_production_org_1" {
+  source             = "../modules/macstadium_go_worker"
+  host_id            = "${module.macstadium_infrastructure.wjb_uuid}"
+  ssh_host           = "${module.macstadium_infrastructure.wjb_ip}"
+  ssh_user           = "${var.ssh_user}"
+  version            = "${var.travis_worker_version}"
+  env                = "production-org-1"
+  index              = "${var.index}"
+  worker_base_config = "${data.template_file.worker_config_common.rendered}"
+  worker_env_config  = "${file("${path.module}/config/travis-worker-production-org-common")}"
+
+  worker_local_config = <<EOF
+export TRAVIS_WORKER_TRAVIS_SITE="org"
+export TRAVIS_WORKER_POOL_SIZE="210"
+export TRAVIS_WORKER_PPROF_PORT="7070"
+export TRAVIS_WORKER_JUPITERBRAIN_ENDPOINT=http://${random_id.jupiter_brain_production_org_token.hex}@127.0.0.1:8081/
+export TRAVIS_WORKER_LIBRATO_SOURCE="travis-worker-production-org-macstadium-$${index}-1-dc18"
```

Comment:
Based on the other code in this file, I'm guessing this should contain `${var.index}` and not `$${index}`.